CHANGES.rst

```diff
@@ -7,6 +7,8 @@ New Features
 Bug Fixes
 ^^^^^^^^^
 
+- Fixed specifying a single value for ``window`` in ``analysis.fit_lines``. [#1164]
```

Comment:
Actually, do you want to add a new test for this particular bug?

Reply:
The existing tests would theoretically have caught it, if they were set to check for the incorrect values. Now they're looking for the values resulting from the fit actually doing the right thing.

Reply:
` yl_double_fit_expected` is the big difference.

Reply:
Gotcha, thanks!